src/components/table-transactions/table-transactions.js

```diff
@@ -2,6 +2,7 @@ import template from "./table-transactions.html";
 import templateRow from "./table-transactions-tr.html";
 
 import { ButtonMoreComponent } from "../button-more/button-more";
+import { ConfirmDialogComponent } from "../confirm-dialog/confirm-dialog";
```

Comment:
please remove confirm dialog from drawer for now -- let it be only in transactions table